scripts/test-tls12-point-format-extension.py

```diff
@@ -0,0 +1,340 @@
+# Author: Alexander Scheel & Hubert Kario, (c) 2022
+# Released under Gnu GPL v2.0, see LICENSE file for details
+"""Test for RFC 8422 / Section 5.2.1 Point Format Extension"""
+
+from __future__ import print_function
+import traceback
+from itertools import chain, islice
+from random import sample
+import sys
+import re
+import getopt
+
+from tlsfuzzer.runner import Runner
+from tlsfuzzer.messages import Connect, ClientHelloGenerator, \
+        ClientKeyExchangeGenerator, ChangeCipherSpecGenerator, \
+        FinishedGenerator, ApplicationDataGenerator, \
+        AlertGenerator, Close
+from tlsfuzzer.expect import ExpectServerHello, ExpectCertificate, \
+        ExpectServerHelloDone, ExpectChangeCipherSpec, ExpectFinished, \
+        ExpectAlert, ExpectClose, ExpectApplicationData, ExpectServerKeyExchange
+from tlslite.extensions import SignatureAlgorithmsExtension, \
+        SignatureAlgorithmsCertExtension, SupportedGroupsExtension, \
+        ECPointFormatsExtension, TLSExtension
+from tlslite.constants import CipherSuite, AlertDescription, \
+        HashAlgorithm, SignatureAlgorithm, ExtensionType, GroupName, \
+        ECPointFormat, AlertLevel, AlertDescription, SignatureScheme
+
+from tlsfuzzer.utils.lists import natural_sort_keys
+from tlsfuzzer.helpers import SIG_ALL
+
+
+version = 1
+
+
+def help_msg():
+    print("Usage: <script-name> [-h hostname] [-p port] [[probe-name] ...]")
+    print(" -h hostname    name of the host to run the test against")
+    print("                localhost by default")
+    print(" -p port        port number to use for connection, 4433 by default")
+    print(" -e probe-name  exclude the probe from the list of the ones run")
+    print("                may be specified multiple times")
+    print(" -n num         run 'num' or all(if 0) tests instead of default(all)")
+    print("                (excluding \"sanity\" tests)")
+    print(" -x probe-name  expect the probe to fail. When such probe passes despite being marked like this")
+    print("                it will be reported in the test summary and the whole script will fail.")
+    print("                May be specified multiple times.")
+    print(" -X message     expect the `message` substring in exception raised during")
+    print("                execution of preceding expected failure probe")
+    print("                usage: [-x probe-name] [-X exception], order is compulsory!")
+    print(" --help         this message")
+
+
+def main():
+    """check if app data records with zero payload are accepted by server"""
```

Comment:
I don't think this applies to this script...
also, in general the `scripts/test-conversation.py` is the better template to use for creating new test scripts